scrollinga.js

```diff
@@ -127,7 +127,10 @@ class Scrollinga {
    * @return {Boolean}
    */
   isScrollDown() {
-    return (this.target.scrollTop + this.target.clientHeight === this.target.scrollHeight);
+    const client = Math.floor((this.target.clientHeight + this.target.scrollTop) / devicePixelRatio);
```

Comment:
Maybe better with `window.devicePixelRation`. Also not IE10 do not have support for it. Maybe a fallback